client/src/app/tabs/EngineProfile.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React, { Fragment } from 'react';
+import classnames from 'classnames';
+
+import { Overlay } from '../../shared/ui';
+import { Fill } from '../slot-fill';
+
+import LinkArrow from '../../../resources/icons/LinkArrow.svg';
+import css from './EngineProfile.less';
+
+
+export function EngineProfile(props) {
+  const { tab } = props;
+
+  const [ open, setOpen ] = React.useState(false, []);
+  const buttonRef = React.useRef(null);
+
+  const engineProfile = tab.meta && tab.meta.engineProfile;
+
+  if (!engineProfile) {
+    return null;
+  }
+
+
+  return (
+    <Fill slot="status-bar__file" group="1_engine">
+      {
+        open &&
+        <EngineProfileOverlay
+          anchor={ buttonRef.current }
+          onClose={ () => setOpen(false) }
+          engineProfile={ engineProfile }
+        />
+      }
+      <button
+        className={ classnames('btn', { 'btn--active': open }) }
+        onClick={ () => setOpen(val => !val) } ref={ buttonRef }
+      >
+        {engineProfile}
+      </button>
+    </Fill>
+  );
+}
+
+function EngineProfileOverlay(props) {
+  const { anchor, onClose, engineProfile } = props;
+
+  return (
+    <Overlay anchor={ anchor } onClose={ onClose }>
+      <Description engineProfile={ engineProfile } />
+    </Overlay>
+  );
+}
+
+function Description(props) {
+  const { engineProfile } = props;
+
+  if (engineProfile === 'Camunda Engine') {
+    return (
+      <Fragment>
+        <Overlay.Body>
+          This diagram is supposed to be executed on <em>Camunda Platform</em>.
+          The properties panel provides the related implementation features.
+          This diagram can be deployed to and started in a connected <em>Camunda Engine</em>.
+        </Overlay.Body>
+        <Overlay.Footer>
+          <Link href="https://docs.camunda.org/manual/latest/">Learn more</Link>
+        </Overlay.Footer>
+      </Fragment>
+    );
+  } else if (engineProfile === 'Zeebe Engine') {
+    return (
+      <Fragment>
+        <Overlay.Body>
```

Comment:
I wonder whether the description body and the link in the footer could be something delivered by the (tab) provider itself instead of doing the distinction here.

Reply:
How would you solve the distinction for body/footer? I don't like the else-if either, but had no better idea to implement this.

Reply:
We would make the tabs provider hardly defining it, so something like

```jsx
// TabsProvider.js
getMeta(file) {
  return {
    engineProfile: { 
      key: 'Camunda Engine',
      description: 'This diagram is supposed ...',
      learnMoreUri: 'https://docs.camunda.io/'
    }
  };
}

// EngineProfile.js
<Fragment>
  <Overlay.Body>
    {engineProfile.description}
  </Overlay.Body>
  <Overlay.Footer>
    <Link href={engineProfile.learnMoreUri}>Learn more</Link>
  </Overlay.Footer>
</Fragment>
```

but that would not help us if we want to have nested content. Otherwise, this would blow up the tabs provider even more.

So I'd be fine with the current implementation.

Reply:
Thanks. Let's keep it as it is and next time we touch the Engine Profile, we can improve it.